docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
       args:
         GOVCMS_PROJECT_VERSION: ${GOVCMS_PROJECT_VERSION:-^1.0}
         DRUPAL_CORE_VERSION: ${DRUPAL_CORE_VERSION:-^8.7}
-        LAGOON_IMAGE_VERSION: ${LAGOON_IMAGE_VERSION_PHP}
+        LAGOON_IMAGE_VERSION: ${LAGOON_IMAGE_VERSION_PHP:-}
```

Comment:
Just checking what this change is since LAGOON_IMAGE_VERSION_PHP was removed from docker-compose.yml in master.

Reply:
Yeah, I'll rebase with master before the merge. It was just to remove some warnings that were annoying me :joy: